src/main/java/bs/dao/Validator.java

```diff
@@ -21,21 +21,21 @@ public static void main(String[] args) {
         System.out.println("RESULT: " + fieldValidator(battleField));
     }
     public static boolean fieldValidator(int[][] field) {
-        List<List<int[]>> ships = new ArrayList<>();
-        List<int[]> found_ship;
+            List<List<int[]>> ships = new ArrayList<>();
+            List<int[]> found_ship;
 
-        for (int i = 0; i < field.length; i++){
-            for (int j = 0; j < field[i].length; j++){
-                if(field[i][j] == 1){
-                    found_ship = findAllParts(i, j, field);
-                    for(int[] ship: found_ship) {
-                        if (Arrays.equals(ship, new int[]{-1, -1})) return false;
+            for (int i = 0; i < field.length; i++){
+                for (int j = 0; j < field[i].length; j++){
+                    if(field[i][j] == 1){
+                        found_ship = findAllParts(i, j, field);
+                        for(int[] ship: found_ship) {
+                            if (Arrays.equals(ship, new int[]{-1, -1})) return false;
```

Comment:
auch wenn nur ein Statement in der If-Abfrage steht bitte geschweifte Klammenr setzen!